packages/mobile/src/pincode/PincodeEnter.tsx

```diff
@@ -28,18 +29,15 @@ interface StateProps {
   currentAccount: string | null
 }
 
-type Props = StateProps & WithTranslation & StackScreenProps<StackParamList, Screens.PincodeEnter>
+type RouteProps = StackScreenProps<StackParamList, Screens.PincodeEnter>
+type Props = StateProps & WithTranslation & RouteProps
 
 class PincodeEnter extends React.Component<Props, State> {
-  static navigationOptions = {
+  static navigationOptions = (navOptions: RouteProps) => ({
+    ...modalScreenOptions(navOptions),
     ...headerWithBackButton,
     gestureEnabled: false,
-    cardStyleInterpolator: ({ current }: StackCardInterpolationProps) => ({
-      containerStyle: {
-        opacity: current.progress,
-      },
-    }),
-  }
```

Comment:
Removed this so that the PIN code screen in modal sheet mode on iOS works correctly, when triggered from the `SendConfirmationModal` screen which is displayed as a modal sheet too on iOS.